charms/mlflow-server/config.yaml

```diff
@@ -2,6 +2,12 @@
 # See LICENSE file for licensing details.
 #
 options:
+  create_default_artifact_root_if_missing:
+    description: |
+      If True, charm will try to create the default_artifact_root bucket in S3 if it does not 
```

Comment:
is there a scenario where this is desirable?

Reply:
I'm not sure.  The case I thought about was if we're in gateway mode (eg: Minio is just a passthrough to storage on say AWS), I could see someone wanting to manage the bucket in the host rather than have this charm try to create things.  But that's also a bit of a stretch so I don't know if it is really needed or not.  What do you think?

Reply:
Yeah sorry I should have said if minio was in gateway mode and used as a passthrough to another blob store.  And yes, we do support/use it (or at least the next customer will).  And no, I did not know it was being deprecated :fearful:

Reply:
Looking at the blocked status message which is very detailed I think this config option is okay as it is.

Reply:
I see. Since we support it, we should keep this config